Source/Exporters/SVGKImage+CGContext.m

```diff
@@ -95,4 +98,47 @@ -(void) renderToContext:(CGContextRef) context antiAliased:(BOOL) shouldAntialia
 	SVGKitLogVerbose(@"[%@] renderToContext: time taken to render CALayers to CGContext (perf improvements:%@): %2.3f seconds)", [self class], perfImprovements, -1.0f * [startTime timeIntervalSinceNow] );
 }
 
+- (void)preprocessRenderingLayerTree:(CALayer *)layerTree {
```

Comment:
Good idea, but please rename the two methods to make it clear that they are NOT an intentional feature of the class, but rather a temporary workaround to deal with one-off rendering issue in Apple code.

(that way they won't get used by people in future thinking that this was the intended class structure).

e.g. something like:

-(void)temporaryWorkaroundPreprocessRenderingLayerTree:


.. AND: copy/paste the // comments up into a doxygen comment for the method declaration, as they explain why the method has to exist.

Reply:
Done.